azure-pipelines.yml

```diff
@@ -32,11 +32,13 @@ stages:
       steps:
       - script: npm install
         displayName: 'Install dependencies'
+      - script: npm install -g pnpm@6.19.0
```

Comment:
What does pnpm bring? I assume this is more for future-proofing than necessary in this instance.

Reply:
The pnpm package manager uses a different type of folder structure to organize the dependencies in the node_modules folder and is more deterministic so I was hoping that it would help identify the issue. (also pnpm is just faster :))